specs/monitoring_integration.adoc

```diff
@@ -0,0 +1,263 @@
+= Spec - Introducing a new component monitoring-integration(1st milestone)
+
+A new component called monitoring-Integration is introduced which integrates Grafana
+with tendrl core.
+
+
+== Problem description
+
+The new monitoring-integration component would primarily focus on dynamic
+configuration of grafana, based on the topology changes in etcd. Some of these
+responsibilities would be:
+
+* Monitoring-integration should create default dashboards in grafana using predefined json.
+  The dashboard json contains details regarding panels, metrics and alerts.
+* Configuring grafana authentication and authorization (https://github.com/Tendrl/specifications/pull/205),
+  based on changes in the primary tendrl AA scheme.
+* Keeping track of the topology changes in etcd to configure the grafana dashboards
+  to ensure that the correct nodes are part of the respective cluster dashboards.
+  (Not planned yet, for 1st milestone grafana is configured using some default
+  configuration from monitoring-integration)
+
+
+== Use Cases
+
+For 1st milestone:
+
+* Monitoring-Integration will create default dashboards in grafana.
+
+For 2nd milestone:
```

Comment:
Heading talks about milestone-1 and you have details about milestone-2 & 3 here. As we talked about in various discussions, take this out from this spec, file specific issues for milestone-2 & 3 items. We need milestone specific specs.